### PR TITLE
using promise.all for selectProgression

### PR DESCRIPTION
**Pull request by chrisdugne**
- PARALLELLISED few fetches during `selectProgression`
- fixed sandbox saveProgression
- added sandbox getNextChapter

### testing 
- [x] Already covered by tests
- [x] Manual testing
- [x] Unit testing

### CODE REVIEW

**Comment by chrisdugne** [comment]
[Mobile app : the game interface lags](https://trello.com/c/rQHtaB9T/1371-mobile-app-the-game-interface-lags)

File: packages/@coorpacademy-player-web/sandbox/fixtures/progressions.js [hunk 1]
```diff
@@ -24,7 +24,8 @@ const getAll = () => {
 };
 
 const save = progression => {
-  return Promise.resolve(progressionStore.set(progression._id, progression));
+  progressionStore.set(progression._id, progression);
+  return Promise.resolve(progression);
```
Comment (chrisdugne) [inline packages/@coorpacademy-player-web/sandbox/fixtures/progressions.js]: ici fix de la sanbbox

**Comment by chrisdugne** [comment]
[performance: enhance selectProgression](https://trello.com/c/kz2FJtE3/1507-performance-enhance-selectprogression)

File: packages/@coorpacademy-player-store/src/actions/ui/progressions.js [hunk 2]
```diff
@@ -62,6 +62,14 @@ export const unselectProgression: SelectAction = {
   }
 };
 
+const fetchData = (dispatch, engine, progressionId, progressionContent): Promise<any> =>
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-player-store/src/actions/ui/progressions.js]: Can this one have the common actions signature ? to be able to call it elsewhere than there?

Comment (chrisdugne) [reply packages/@coorpacademy-player-store/src/actions/ui/progressions.js]: ok why not, but it's sure it won't be reused elsewhere :) 
https://github.com/CoorpAcademy/components/pull/1618/commits/7d7752d409fb1f233e497b0da0ce2ac414b743f3

**Pull request merged**